start-site/src/main/resources/application.yml

```diff
@@ -105,8 +105,15 @@ initializr:
         mappings:
           - compatibilityRange: "[2.0.0.RELEASE,2.0.x.BUILD-SNAPSHOT]"
             version: 2.0.3.RELEASE
-          - compatibilityRange: "2.1.0.RELEASE"
+          - compatibilityRange: "[2.1.0.RELEASE,2.1.10.RELEASE]"
             version: 2.1.4.RELEASE
+          - compatibilityRange: "[2.1.10.RELEASE,2.1.x.BUILD-SNAPSHOT]"
+            version: 3.0.1.RELEASE
+          - compatibilityRange: "[2.1.x.BUILD-SNAPSHOT,2.2.0.M4]"
+            version: 3.0.2.BUILD-SNAPSHOT
+            repositories: spring-snapshots,spring-milestones
+          - compatibilityRange: "[2.2.0.M4,2.2.x.BUILD-SNAPSHOT]"
```

Comment:
`2.2.0.RELEASE` is out so I think we can go ahead with ignoring that `M4`; This would have been relevant (and useful) when Spring Boot 2.2 was in milestone phase.

Reply:
Ok np! I was copying from spring-cloud section above. ;)